### PR TITLE
Clean PKGBUILD

### PR DESCRIPTION
**Pull request by iTrooz**
As discussed on Discord : https://discord.com/channels/789833418631675954/789833418631675957/1016713453030887524

@Foxboron if you want to take a look ^^

### CODE REVIEW

File: dist/Arch/PKGBUILD [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # Contributor: Morten Linderud <foxboron@archlinux.org>
 
 pkgname=imhex-bin
-pkgver=%version%
-pkgrel=1
+pkgver=1.22.0
```
Comment (WerWolv) [inline dist/Arch/PKGBUILD:5]: I'd really like to avoid having to update the version in here every time

**Review by WerWolv — CHANGES_REQUESTED** [review]

File: dist/Arch/PKGBUILD (hunk 1 above)
Comment (iTrooz) [reply dist/Arch/PKGBUILD]: OOPS, really sorry, I forgot about it, I did it to test the PKGBUILD

**Review by WerWolv — APPROVED** [review]

**Comment by WerWolv** [comment]
Thank you!

**Comment by Foxboron** [comment]
Looks fine, thanks @iTrooz :)

**Pull request merged**